Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/eventregistration/attendee/checkins/CheckInsFragment.kt

```diff
@@ -125,6 +127,13 @@ class CheckInsFragment : Fragment(R.layout.trace_location_attendee_checkins_frag
         }
     }
 
+    private fun showInvalidQrCodeInformation(@StringRes errorTextRes: Int) =
+        AlertDialog.Builder(requireContext()).apply {
```

Comment:
Please MaterialBuilder?

Reply:
Should we also change this for the other AlertBuilders in this Fragment?

Reply:
That would be great 👍